src/components/DocNode/DocNode.tsx

```diff
@@ -75,101 +72,133 @@ type DocNodeFieldProps = {
 const DocNodeField: FC<DocNodeFieldProps> = ({ title, description }) => {
   return (
     <tr>
-      <td className={classes.descTitle}>{title}</td>
-      <td className={classes.desc}>{description}</td>
+      <td className={styles.descTitle}>{title}</td>
+      <td className={styles.desc}>{description}</td>
     </tr>
   );
 };
 
-type DocGuidanceProps = {
-  pathwayState: GuidanceState;
-};
+function renderBranch(documentation: DocumentationResource | undefined): ReactElement[] {
+  const returnElements: ReactElement[] = [];
 
-const DocGuidance: FC<DocGuidanceProps> = ({ pathwayState }) => {
-  const resource = pathwayState.action[0].resource;
-  const coding =
-    'medicationCodeableConcept' in resource
-      ? resource.medicationCodeableConcept.coding
-      : resource.code.coding;
-  return (
-    <>
-      <DocNodeField key="Notes" title="Notes" description={pathwayState.action[0].description} />
-      <DocNodeField key="Type" title="Type" description={resource.resourceType} />
+  if (documentation?.resource) {
+    switch (documentation.resourceType) {
+      case 'Observation': {
+        const observation = documentation.resource as fhir.Observation;
+
+        const valueCoding = observation.valueCodeableConcept?.coding;
+        valueCoding &&
+          returnElements.push(
+            <DocNodeField
+              key="ValueSystem"
+              title="System"
+              description={
+                <>
+                  {valueCoding[0].system}
+                  <a href={valueCoding[0].system} target="_blank" rel="noopener noreferrer">
+                    <FontAwesomeIcon icon="external-link-alt" className={styles.externalLink} />
+                  </a>
+                </>
+              }
+            />,
+            <DocNodeField key="ValueCode" title="Code" description={valueCoding[0].code} />,
+            <DocNodeField key="ValueDisplay" title="Display" description={valueCoding[0].display} />
+          );
+
+        const date = observation.effectiveDateTime;
```

Comment:
This was moved from another part of the file. I think it came from Keeyan originally and he probably chose DateTime because that is the Synthea output.

Reply:
Created [PATHWAYS-178](https://jira.mitre.org/browse/PATHWAYS-178)